docs/designers-developers/developers/tutorials/javascript/js-build-setup.md

```diff
@@ -148,20 +148,19 @@ Likewise, you do not need to include `node_modules` or any of the above configur
 
 ### Dependency Management
 
-The build step also produces a `index.deps.json` file that contains a JSON array of the wp packages required for your block. For our simple example above would be `["wp-blocks","wp-polyfill"]`
+Using wp-scripts ver 5.0.0+ build step will also produce a `index.asset.php` file that contains an array of dependencies and version number for your block. For our simple example above is something like:
```

Comment:
It generates this `.asset.php` file for each of the entry points. so if you add your cutsom entry points they will get this file swell :)

Reply:
That is true, but our JS tutorial is intended for beginners and does not include any description on how one would configure multiple entry points. It is more of an advanced topic better left for package documentation not this tutorial.